### DISCUSSION TITLE
How to draw text on an image in arrayfire::Window?

### DISCUSSION THREAD

**Discussion by 3togo** [arrayfire/arrayfire-rust#351] [Q&A] [answered]
How to draw text on an image in arrayfire::Window?

**Comment by 9prady9** [comment, marked as answer]
Random text is not possible, but a title for each cell in the grid layout is possible currently. See the below example on how to do it
https://arrayfire.org/arrayfire-rust/arrayfire/struct.Window.html#examples

https://github.com/arrayfire/arrayfire-rust/blob/master/examples/histogram.rs

**Comment by 3togo** [comment]
@9prady9 

many thanks
